.travis/linux/deploy.sh

```diff
@@ -25,6 +25,9 @@ mv squashfs-root linuxdeployqt
 
 linuxdeployqt/AppRun "$APPDIR/usr/share/applications/dwarftherapist.desktop" -extra-plugins=styles/libfusiondark.so,styles/libDarkStyle.so -bundle-non-qt-libs -qmake=$QT_PREFIX/bin/qmake
 
+rm "$APPDIR/AppRun"
+wget "https://github.com/AppImage/AppImageKit/releases/download/continuous/AppRun-x86_64" -O "$APPDIR/AppRun"
```

Comment:
```suggestion
wget "https://github.com/AppImage/AppImageKit/releases/download/continuous/AppRun-x86_64" -O "$APPDIR/AppRun"
chmod +x "$APPDIR/AppRun"
```
I don't know much about App Image stuff but I think this has to be executable, see my last comment on #215

Reply:
Thanks. It is fixed now and tested on travis (test build here: https://github.com/cvuchener/Dwarf-Therapist/releases/tag/untagged-1654123c24ff33b6ea5d).